app/script/entity/message/Message.coffee

```diff
@@ -226,3 +231,33 @@ class z.entity.Message
     else if @status() isnt updated_status
       return @status updated_status
     return false
+
+  start_ephemeral_timer: =>
+    return if @ephemeral_timer
+    @ephemeral_remaining new Date(@ephemeral_expires()) - Date.now()
```

Comment:
i am not a fan of moving the localization inside the message, since this is what a component is for